pipeline/pipelines/gapic_generation_pipeline.py

```diff
@@ -98,7 +91,7 @@ def _get_gapic_package_tasks(self, **kwargs):
 class _NodeJSGapicTaskFactory(GapicTaskFactoryBase):
 
     def _get_gapic_package_tasks(self, **kwargs):
-        return [gapic_tasks.GapicMergeTask('GapicMerge', inject=kwargs),
+        return [gapic_tasks.GapicCopyTask('GapicCopy', inject=kwargs),
                 gapic_tasks.GapicPackmanTask('GapicPackman', inject=kwargs)]
 
 
```

Comment:
Is there a good reason to keep the CSharp task as it is? Or can we make it use the default copy step? (Because the default previously was a merge step, it didn't seem appropriate to change CSharp to use it)